Cannot find module log4js with TypeScript

log4js installed as dependency through npm.

tsconfig.json using module:esnext
package.json using type:module

Not working : 
```typescript
import log4js = require("log4js");
//Import assignment cannot be used when targeting ECMAScript modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'import d from "mod"', or another module format instead.
import * as log4js from "log4js";
//Cannot find module 'log4js'
```

Current workaround:
```typescript
import * as log4js from "../../../node_modules/log4js/types/log4js";
```

--- a/README.md
+++ b/README.md
@@ -99,16 +99,15 @@ There's also [an example application](https://github.com/log4js-node/log4js-exam
 ## TypeScript
 
 ```ts
-import { configure, getLogger } from "log4js";
-configure("./filename");
-const logger = getLogger();
-logger.level = "debug";
-logger.debug("Some debug messages");
-
-configure({
+import log4js from "log4js";
+log4js.configure({
   appenders: { cheese: { type: "file", filename: "cheese.log" } },
   categories: { default: { appenders: ["cheese"], level: "error" } }
 });
+
+const logger = log4js.getLogger();
+logger.level = "debug";
+logger.debug("Some debug messages");
 ```
 
 ## Contributing

